spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -179,21 +200,21 @@
               },
               {
                 "title" => "Invalid study_mode",
-                "detail" => "Study mode can't be blank",
+                "detail" => "You need to pick an option",
```

Comment:
Should these not be 'you need to pick a[n] [thing]' like we have with age range and level?

Reply:
Taken from the content that Paul G wrote - he went with "An option" here